frontend/src/store/modules/likes.js

```diff
@@ -1,3 +1,5 @@
+import itemsApi from "../../api/items";
```

Comment:
Where is this being used?

Reply:
inside upsertLike action
it is using itemsApi.likeItem